src/sentry/snuba/entity_subscription.py

```diff
@@ -375,13 +357,50 @@ def aggregate_query_results(
 class MetricsCountersEntitySubscription(BaseMetricsEntitySubscription):
     entity_key: EntityKey = EntityKey.MetricsCounters
     metric_key: SessionMRI = SessionMRI.SESSION
-    aggregation_func: str = "sum"
+
+    def get_aggregations(self) -> List[List[Optional[str]]]:
+        session_status_crashed = resolve(self.org_id, "crashed")
+        session_status_init = resolve(self.org_id, "init")
+        return [
+            [
+                f"sumIf(value, equals({self.session_status}, {session_status_init}))",
+                None,
+                "count",
+            ],
+            [
+                f"sumIf(value, equals({self.session_status}, {session_status_crashed}))",
+                None,
+                "crashed",
+            ],
+        ]
+
+    def get_extra_conditions(self) -> List[List[Any]]:
+        crashed = resolve(self.org_id, "crashed")
+        init = resolve(self.org_id, "init")
+        return [[self.session_status, "IN", [crashed, init]]]
 
 
 class MetricsSetsEntitySubscription(BaseMetricsEntitySubscription):
     entity_key: EntityKey = EntityKey.MetricsSets
     metric_key: SessionMRI = SessionMRI.USER
-    aggregation_func: str = "uniq"
+
+    def get_aggregations(self) -> List[List[Optional[str]]]:
+        session_status_crashed = resolve(self.org_id, "crashed")
+        return [
+            [
+                "uniq(value)",
+                None,
+                "count",
+            ],
+            [
+                f"uniqIf(value, equals({self.session_status}, {session_status_crashed}))",
+                None,
+                "crashed",
+            ],
+        ]
+
+    def get_extra_conditions(self) -> List[List[Any]]:
+        return []
```

Comment:
For the *user* count, we now query the metric over all statuses, so we cannot filter down to status.